roles/tasmota-power-script/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Playbook to implement a remote startup script for users
+
+- name: install script dependencies
+  apt:
+    name:
+      - dialog
+      - curl
+    state: present
+
+- name: Make sure user is present on host
+  user:
+    name: startup_script
+    shell: /usr/bin/startup_script.sh
+    state: present
+    password: $1$ZBQbTRIM$/6OLvzsHYqHl4eszgo0hk.
+    create_home: no
+
+- name: copy the template over
+  template:
+    src: startup_script.sh
+    dest: /usr/bin/startup_script.sh
+    owner: nobody
+    group: nogroup
```

Comment:
Owner und Group: root. Es gibt Prozesse die als nobody laufen, weil sie dann wenig Rechte haben. Diese Prozesse sollen gerade nicht in der lage sein, diese Skript zu modfifizieren.

Reply:
Korrigiert.